pkg/container/container.go

```diff
@@ -62,6 +72,18 @@ type Container interface {
 	// SelinuxLabel returns the container's SelinuxLabel
 	// it takes the sandbox's label, which it falls back upon
 	SelinuxLabel(string) ([]string, error)
+
+	// spec functions
+
+	// returns the spec
+	Spec() *generate.Generator
+
+	// SpecAddMount adds a mount to the container's spec
+	// it takes the destination, the mount type, the source type, the mount to remove, and the options
```

Comment:
> the mount to remove

looks like this comment is no longer valid

Reply:
Well, it might make sense to say instead that if there's another mount with the same destination, it is removed.

Reply:
fixed !